### PR TITLE
Remove custom batch normalization layer.

### PR DESCRIPTION
**Pull request by hgaiser**
Setting the `trainable` flag now also disables all updates for the BatchNormalization layer.

Source: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/layers/normalization_v2.py

### CODE REVIEW

**Review by vcarpani — APPROVED** [review]
That's a good one ;)
